packages/organizations/origin-organization-irec-api/src/connection/handlers/create-connection.handler.ts

```diff
@@ -129,8 +130,11 @@ export class CreateConnectionHandler implements ICommandHandler<CreateConnection
 
         for (const account of accounts) {
             if (!irecAccounts.some((a) => a.type === account)) {
+                const numberOfChars = 4;
+                const randomCode = randomBytes(numberOfChars / 2).toString('hex'); // 1 byte is 2 characters
+
                 await this.irecService.createAccountByTokens(clientId, clientSecret, tokens, {
-                    code: `${account}-account`,
+                    code: `${account}-${randomCode}`,
```

Comment:
it would probably not be unique at some point, is that correct?

Reply:
Well, the original problem was that it was not unique. Not it will be. We have 16 characters in hex, which gives us 65k combinations. Should be more than sufficient. If it's not, then we can always increase this.

If account is not unique it will be rejected by irec. In such case you can just retry to get another combination, and if this happens too often, then devs can just increase this value

So in IREC response i got info that it would be nice if account code is easy to copy or write manually, that's why I went with 4 random chars only.